src/dbnode/persist/fs/write.go

```diff
@@ -146,12 +146,11 @@ func NewWriter(opts Options) (DataFileSetWriter, error) {
 // opening / truncating files associated with that shard for writing.
 func (w *writer) Open(opts DataWriterOpenOptions) error {
 	var (
-		err             error
-		namespace       = opts.Identifier.Namespace
-		shard           = opts.Identifier.Shard
-		blockStart      = opts.Identifier.BlockStart
-		volumeIndex     = opts.Identifier.VolumeIndex
-		nextVolumeIndex = opts.Identifier.VolumeIndex
```

Comment:
This was the result of a bad merge, causing duplicate variables for the same thing.